ui/mirage/handlers/sync.js

```diff
@@ -14,7 +14,7 @@ export const associationsResponse = (schema, req) => {
   const records = schema.db.syncAssociations.where({ type, name });
   const associations = records.length
     ? records.reduce((associations, association) => {
-        const key = `${association.mount}/${association.secret_name}`;
+        const key = `${association.mount}_12345/${association.secret_name}`;
```

Comment:
is `_12345` desired here? Curious why we would hardcode that here instead of updating the `mount` value returned on subkeys?

Reply:
Good question. We were originally under the assumption that the key in the response was going to be `${mount}/${secret_name}` but it turns out that it is actually `${accessor}/${secret_name}/${sub_key}. We are setting the id for the record as `${mount}/${secret_name}` so I updated all the keys in mirage to deliberately not match the `mount` value. Since we use the `associationsResponse` in tests I wanted to be sure that the adapter was returning the correct record and updating the keys ensures that the id error is no longer an issue.